src/amo/components/AddonFeedbackForm/index.js

```diff
@@ -130,6 +138,37 @@ export class AddonFeedbackFormBase extends React.Component<InternalProps> {
               </div>
 
               <AddonTitle addon={addon} />
+
+              {!this.isAddonNonPublic() && (
```

Comment:
Shouldn't we show _something_ for non-public add-ons?  Like the guid (or whatever was provided as the add-on ID in the url)

Reply:
We do, currently. It's in the `AddonTitle` component above. That said, UX wants us to get rid of the entire (content) header in https://github.com/mozilla/addons-frontend/issues/12537